include/EASTL/hash_set.h

```diff
@@ -117,8 +117,19 @@ namespace eastl
 		/// hash_set
 		///
 		/// Default constructor.
-		/// 
-		explicit hash_set(const allocator_type& allocator = EASTL_HASH_SET_DEFAULT_ALLOCATOR)
+		///
+		hash_set()
+			: base_type(0, Hash(), mod_range_hashing(), default_ranged_hash(), Predicate(), eastl::use_self<Value>(), EASTL_HASH_SET_DEFAULT_ALLOCATOR)
```

Comment:
Same as above, reduce code duplication by forwarding to the constructor taking an allocator.